spring-cloud-stream-app-starters-docs/src/main/asciidoc/overview.adoc

```diff
@@ -1,8 +1,185 @@
 [[overview]]
-== Spring Cloud Stream Modules 
 
-This section goes into more detail about how you can work with Spring Cloud Stream Module as standalone applications or with Spring Cloud Data Flow.
+This section will provide you with a detailed overview of Spring Cloud Stream Application Starters, their purpose, and how to use them.
+It assumes familiarity with general Spring Cloud Stream concepts, which can be found in its reference documentation.
 
-=== Overview
+== Introduction
 
-TBD
+Spring Cloud Stream Application Starters provide you with predefined Spring Cloud Stream applications that you can run independently or with Spring Cloud Data Flow.
+You can also use the starters as basis for creating your own applications.
+They include:
+
+* connectors (sources and sinks) for middleware including message brokers, storage (relational, non-relational, filesystem);
+* adapters for various network protocols;
+* generic processors that can be customized via SpEL or scripting.
+
+You can find a detailed listing of all the starters, as well as their options in the <<starters,corresponding>> section of this guide.
+
+=== Starters and pre-built applications
+
+As a user of Spring Cloud Stream Application Starters you have access to two types of artifacts.
+
+_Starters_ are libraries that contain the complete configuration of a Spring Cloud Stream application with a specific role (e.g. an _http source_ that receives HTTP POST requests and forwards the data on its output channel to downstream Spring Cloud Stream applications).
+Starters are not executable applications, and are intended be included in other Spring Boot applications, along with a Binder implementation.
+
+_Prebuilt applications_ are Spring Boot applications that include the starters and a binder implementation.
+Prebuilt applications are uberjars and include minimal code required to execute standalone.
+For each starter, the project provides a prebuilt version including the Kafka binder, as well as one including the Rabbit MQ binder.
```

Comment:
“For each starter, the project provides a prebuilt version including the Kafka binder and a prebuilt version including the RabbitMQ binder.”